### PR TITLE
In-Plugin Editing - Image Caption

### PR DESCRIPTION
**Pull request by mharsat**

### CODE REVIEW

File: packages/plugin-image/web/src/InPluginInput.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  );
+
+  render() {
+    const { onChange, value } = this.props;
+    return (
+      <input
+        className={this.className}
+        value={value}
+        readOnly={!onChange}
+        onChange={onChange}
```
Comment (shaulgo) [inline packages/plugin-image/web/src/InPluginInput.jsx]: extract the value from the event here

File: packages/plugin-image/web/statics/styles/in-plugin-input.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.inPluginInput {
+    border: none;
```
Comment (shaulgo) [inline packages/plugin-image/web/statics/styles/in-plugin-input.scss]: add dir: auto

File: packages/plugin-image/web/statics/styles/in-plugin-input.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    line-height: 21px;
+    width: 100%;
+    position: inherit;
+    z-index: 1;
+    font: inherit;
+    font-size: 14px;
+
+    &Viewer {
+        @extend .inPluginInput;
```
Comment (shaulgo) [inline packages/plugin-image/web/statics/styles/in-plugin-input.scss]: do you need @extend?

File: packages/plugin-image/web/src/InPluginInput.jsx (hunk 1 above)
Comment (mharsat) [reply packages/plugin-image/web/src/InPluginInput.jsx]: shouldn't the value be taken from the `componentData`?

File: packages/plugin-image/web/statics/styles/in-plugin-input.scss (hunk 3 above)
Comment (mharsat) [reply packages/plugin-image/web/statics/styles/in-plugin-input.scss]: I want the input area to have the normal clicking cursor in viewer so it won't look editable. I need the extend in order to have both `inPluginInput` and `inPluginInputViewer` classes without repeating code. An alternative is to extract `cursor: pointer` into a different class

File: packages/plugin-image/web/statics/styles/in-plugin-input.scss (hunk 2 above)
Comment (mharsat) [reply packages/plugin-image/web/statics/styles/in-plugin-input.scss]: nice!

**Pull request merged**